src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  let tempState = state;
```

Comment:
This is unnecessary, as you are assigning here the reference to the state object so you are opereting on a state object.

Reply:
Ok understandable